packages/twenty-front/src/modules/users/components/UserProviderEffect.tsx

```diff
@@ -31,6 +33,9 @@ export const UserProviderEffect = () => {
   const setCurrentWorkspaceMember = useSetRecoilState(
     currentWorkspaceMemberState,
   );
+  const setCurrentWorkspaceMembers = useSetRecoilState(
```

Comment:
"CurrentWorkspaceMembers" is confusing imho. If we are talking about multiple different workspace members I wouldn't use the term "current" here